Mathlib/LinearAlgebra/Matrix/Zpow.lean

```diff
@@ -308,19 +314,21 @@ theorem Commute.mul_zpow {A B : M} (h : Commute A B) : ∀ i : ℤ, (A * B) ^ i
   | (n : ℕ) => by simp [h.mul_pow n, -mul_eq_mul]
   | -[n+1] => by
     rw [zpow_negSucc, zpow_negSucc, zpow_negSucc, mul_eq_mul _⁻¹, ← mul_inv_rev, ← mul_eq_mul,
-      h.mul_pow n.succ, (h.pow_pow _ _).Eq]
+      h.mul_pow n.succ, (h.pow_pow _ _).eq]
 #align matrix.commute.mul_zpow Matrix.Commute.mul_zpow
 
+set_option linter.deprecated false in
 theorem zpow_bit0' (A : M) (n : ℤ) : A ^ bit0 n = (A * A) ^ n :=
   (zpow_bit0 A n).trans (Commute.mul_zpow (Commute.refl A) n).symm
 #align matrix.zpow_bit0' Matrix.zpow_bit0'
 
+set_option linter.deprecated false in
 theorem zpow_bit1' (A : M) (n : ℤ) : A ^ bit1 n = (A * A) ^ n * A := by
   rw [zpow_bit1, Commute.mul_zpow (Commute.refl A)]
 #align matrix.zpow_bit1' Matrix.zpow_bit1'
 
 theorem zpow_neg_mul_zpow_self (n : ℤ) {A : M} (h : IsUnit A.det) : A ^ (-n) * A ^ n = 1 := by
-  rw [zpow_neg h, mul_eq_mul, nonsing_inv_mul _ (h.det_zpow _)]
+  rw [zpow_neg h, mul_eq_mul, nonsing_inv_mul _ (IsUnit.det_zpow h _)]
```

Comment:
Does dot notation really not work here?

Reply:
After changing the lemma above it does now work in all places.